src/NexusMods.App.UI/Windows/MainWindowViewModel.cs

```diff
@@ -19,6 +19,7 @@
 using NexusMods.App.UI.WorkspaceSystem;
 using ReactiveUI;
 using ReactiveUI.Fody.Helpers;
+using Tmds.DBus.Protocol;
 
 namespace NexusMods.App.UI.Windows;
```

Comment:
This shouldn't be in .Windows. Instead we want it in .Linux, and if we have to pull in a `DBus` dependency for this as well, we probably want it in a separate project (like we discussed with the Linux/Windows/OSX split

Reply:
This is `NexusMods.App.UI.Windows`, the `Windows` is not referring to the OS but to the concept of windows, this is the `MainWindowViewModel.cs` file.